spec/rails_helper.rb

```diff
@@ -50,5 +50,10 @@
     driven_by(:rack_test)
   end
 
+  config.before(:each, type: :feature) do
+    create(:academic_cycle, :current)
```

Comment:
This is already about to be introduced in this PR https://github.com/DFE-Digital/register-trainee-teachers/pull/2460/files#diff-053aa3ea46cb7c6a649e0d0fc592ce729b66e1d44be475516fd58857ba695862